lib/src/builders/annotation.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/analyzer.dart';
+import 'package:code_builder/src/builders/class.dart';
+import 'package:code_builder/src/builders/parameter.dart';
+import 'package:code_builder/src/builders/shared.dart';
+
+/// Lazily builds an [Annotation] AST when [buildAnnotation] is invoked.
```

Comment:
I'd remove "Lazily" from the comments throughout the package. This word only makes me wonder what it means and might scare me away from using this class :)